StudioCore/MsbEditor/SceneTree.cs

```diff
@@ -735,9 +748,29 @@ public void OnGui()
                             {
                                 _selection.ClearSelection();
                                 _editorActionManager.Clear();
-                                pendingUnload = m;
+                                _universe.UnloadContainer(m);
+                                GC.Collect();
+                                GC.WaitForPendingFinalizers();
```

Comment:
We shouldn't need to manually trigger the GC as it causes long hangs

Reply:
Just to make sure we're on the same page, I didn't add any additional garbage collection, it was just moved. See https://github.com/soulsmods/DSMapStudio/pull/443/files#diff-4c594b25e8e69ac23ce63450678107bf1b30601051f8d330515526ae85fc27afL851

Let me know if you think those instances should be removed.